docs/migrating-to-emotion-10.mdx

```diff
@@ -71,7 +71,7 @@ import styled, { css } from 'react-emotion'
 // ↓ ↓ ↓ ↓ ↓ ↓
 
 import styled from '@emotion/styled'
-import { css } from 'emotion'
```

Comment:
Actually, I think this was correct - it matches the code [here](https://github.com/emotion-js/emotion/blob/62dd16b645a55687ff094487e6c6319e1dd69f95/packages/eslint-plugin/src/rules/import-from-emotion.js#L32). The difference is really nuanced though 🤷‍♂️ As part of the incremental migration you were able to provide the cache from `'emotion'` package - that's because in Emotion 9 actual class names were returned from `css` and it wasn't an uncommon situation to leverage that.

Reply:
Thanks for the note! I think this may not have worked as expected for me because I skipped straight to v11, but used the v10 docs to help with migration — it looks like the packages were renamed in 11, so that's likely where I got tripped up.

Appreciate the clarification :)